src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -107,36 +107,37 @@ const ArtworkAuctionCreateAlertHeader: FC<ArtworkAuctionCreateAlertHeaderProps>
       entity={entity}
       artistSlug={artistSlug}
     >
-      <GridColumns py={6}>
+      <GridColumns py={[4, 4, 6]}>
         <Column span={12}>
-          <Text variant="lg" textAlign={["left", "center"]}>
+          <Text variant={["md", "md", "lg"]} textAlign={["left", "center"]}>
             Bidding for <i>{artwork.title?.trim()}</i>
             {artistName} has ended.
           </Text>
           <Text
-            variant={["sm-display", "lg"]}
+            variant={["sm-display", "md", "lg"]}
             textAlign={["left", "center"]}
             textColor={["black60", "black100"]}
           >
             Get notified when similar works become available.
           </Text>
         </Column>
-
-        <Column span={2} start={6}>
-          <ArtworkSidebarCreateAlertButtonFragmentContainer artwork={artwork} />
-        </Column>
-
         <Column span={12}>
-          <Media greaterThan="xs">
-            <Spacer y={4} />
-            <ArtworkAuctionCreateAlertTooltip />
+          <Media lessThan="sm">
+            <Spacer y={1} />
           </Media>
+          <Box mx="auto" width={["100%", 209]}>
+            <ArtworkSidebarCreateAlertButtonFragmentContainer
+              artwork={artwork}
+            />
+          </Box>
+        </Column>
+        <Column span={12} display={["none", "block"]}>
+          <Spacer y={4} />
+          <ArtworkAuctionCreateAlertTooltip />
         </Column>
 
-        <Column span={12}>
-          <Media greaterThan="xs">
-            <SuggestedArtworksShelfQueryRenderer {...criteria} />
-          </Media>
+        <Column span={12} display={["none", "block"]}>
```

Comment:
We need to remove them like this instead of `Media` because we had empty Columns on mobile.